Sources/TuistSigning/SigningInstaller.swift

```diff
@@ -1,37 +1,29 @@
 import Foundation
 import TSCBasic
+import TuistCore
 import TuistSupport
 
-enum SigningInstallerError: FatalError, Equatable {
-    case noFileExtension(AbsolutePath)
-    case provisioningProfilePathNotFound(ProvisioningProfile)
-    case revokedCertificate(Certificate)
-    case expiredProvisioningProfile(ProvisioningProfile)
-
-    var type: ErrorType {
-        switch self {
-        case .noFileExtension, .provisioningProfilePathNotFound, .revokedCertificate, .expiredProvisioningProfile:
-            return .abort
-        }
+extension LintingIssue {
+    static func noFileExtension(_ path: AbsolutePath) -> Self {
+        Self(reason: "Unable to parse extension from file at \(path.pathString)", severity: .error)
     }
 
-    var description: String {
-        switch self {
-        case let .noFileExtension(path):
-            return "Unable to parse extension from file at \(path.pathString)"
-        case let .revokedCertificate(certificate):
-            return "The certificate has been revoked \(certificate.name)"
-        case let .expiredProvisioningProfile(provisioningProfile):
-            return "The provisioning profile \(provisioningProfile.name) has expired"
-        case let .provisioningProfilePathNotFound(provisioningProfile):
-            return "Could not find any path for \(provisioningProfile.name)"
-        }
+    static func expiredProvisioningProfile(_ profile: ProvisioningProfile) -> Self {
+        Self(reason: "The provisioning profile \(profile.name) has expired", severity: .warning)
```

Comment:
we'd like to still be able to work when this happens

Reply:
I'd suggest adding another sentence along the lines of:
```
Bear in mind that attempting to export or run the app on a device might lead to an error.
```